ex03.ex

```diff
@@ -55,7 +55,16 @@ defmodule Ex03 do
 
   """
 
-  def odd_even . . . "your code"
+  def atom_Generator(i), do: 
```

Comment:
In the end, I deducted 1 for using an `if`, but added 1 back for the nice style of using `atom_generator`